internal/statestore/redis.go

```diff
@@ -170,16 +170,7 @@ func (rb *redisBackend) CreateTicket(ctx context.Context, ticket *pb.Ticket) err
 		return status.Errorf(codes.Internal, "%v", err)
 	}
 
-	value, err := proto.Marshal(ticket)
-	if err != nil {
-		redisLogger.WithFields(logrus.Fields{
-			"key":   ticket.GetId(),
-			"error": err.Error(),
-		}).Error("failed to marshal the ticket proto")
-		return status.Errorf(codes.Internal, "%v", err)
-	}
-
-	err = redisConn.Send("SET", ticket.GetId(), value)
+	err = redisConn.Send("SET", ticket.GetId(), proto.MarshalTextString(ticket))
```

Comment:
This will "serialize" using protobuf's text format, which has a number of large disadvantages:
- It's slower.
- It's much larger.
- I think unrecognized fields are dropped.
- Field renames aren't forward/backwards compatible.

You should be instead be serializing the proto to bytes, then serializing those bytes into a string which works on redis, eg https://golang.org/pkg/encoding/base64/